cinder/README.md

```diff
@@ -0,0 +1,70 @@
+# CSI Cinder driver
+
+## Usage:
+
+### Start Cinder driver
+```
+$ export OS_USERNAME=admin
+$ export OS_PASSWORD=password
```

Comment:
Is there a way to hide the password? This will become a security hole.

Reply:
Hi Chakri, by [this link]( https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloudconf). It stores password in conf file. How do you think if we need to add these env into extra config file or keep them in env? Looking forward your suggestions.

Reply:
@edisonxiang how about storing it in a config file and pass the config file? In environments like K8S, config file can be made a secret and passed to the driver securely.

Reply:
Thanks @chakri-nelluri I will try.